native/explorer/cargo-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly-2023-06-23"
```

Comment:
I forgot to call this out earlier. The most recent rust nightly didn't work for me and I had to use the specific version from the README. I've never really used rust before and IDK if what I did is a good practice. 🤷‍♂️

Just wanted to make sure this wasn't lost in the shuffle.

Reply:
No problem! I think it's a good idea to keep this file. Polars is doing the same.
I think we can simplify our CI files to read from this, but I don't know how yet.

Thanks for pointing this out!